includes/Core/Authentication/Authentication.php

```diff
@@ -1105,7 +1107,16 @@ private function get_reauthentication_needed_notice() {
 					ob_start();
 					?>
 					<p>
-						<?php esc_html_e( 'Site Kit by Google: You need to reauthenticate your Google account.', 'google-site-kit' ); ?>
+						<?php
+							echo esc_html(
+								sprintf(
+									/* translators: %1$s: Plugin name. %2$s: Notice content. */
+									__( '%1$s: %2$s', 'google-site-kit' ),
+									get_plugin_data( GOOGLESITEKIT_PLUGIN_MAIN_FILE )['Name'],
+									esc_html__( 'You need to reauthenticate your Google account.', 'google-site-kit' )
```

Comment:
We don't need to escape HTML here as well since this is being done on the whole output above.
```suggestion
									__( 'You need to reauthenticate your Google account.', 'google-site-kit' )
```

Reply:
Apologies for missing that. Sorted.